src/universal/containers/Logout/LogoutContainer.js

```diff
@@ -0,0 +1,29 @@
+import React, {PropTypes, Component} from 'react';
+import {connect} from 'react-redux';
+import {push} from 'react-router-redux';
+import {show} from 'universal/modules/notifications/ducks/notifications';
+import {removeAuthToken} from 'universal/redux/authDuck';
+
+const logoutSuccess = {
+  title: 'Tootles!',
+  message: 'You\'ve been logged out successfully.',
+  level: 'success'
+};
+
+@connect()
+export default class LogoutContainer extends Component {
+  static propTypes = {
+    dispatch: PropTypes.func.isRequired
+  };
+
+  componentWillMount() {
+    const {dispatch} = this.props;
+    dispatch((thunkDispatch) => Promise.all([
```

Comment:
Yes, that should be the case. However, something funny is going on here that deserves code-comment. When the router `push('/')` happens, the router will route and the as a consequence, the store will be persisted and re-created. When that happens, it's got some nasty side effects including not getting to the next action after the `push('/')`. I've found the thunk avoids this. Not great. What do you think we should do? Comment? Refactor somehow?